Automattic/wp-calypso
# Add happychat client npm package to mobile chat page

**Built on top of https://github.com/Automattic/wp-calypso/pull/23842**
## Summary
This PR uses the in-development `happychat-client` npm package to replace the happychat entry point used for mobile user at the route `me/chat`

**This will not be merged until we publish the npm package**

## Testing
0. Make sure you use the same npm version for both projects
1. Switch happychat-client branch to `add/layout-option` https://github.com/Automattic/happychat-client/pull/196
2. `npm link` in happychat-client
3. `npm link happychat-client` in calypso
4. Run `npm run dev:npm` in happychat-client and leave it running
5. Start calypso
6. Access `/me/chat` chat should be loaded and it should work properly
